cli/src/commands/util/exec.rs

```diff
@@ -59,6 +58,9 @@ use crate::ui::Ui;
 /// # This last empty string will become "$0" in bash, so your actual arguments
 /// # are all included in "$@" and start at "$1" as expected.
 /// ```
+///
+/// > Note: Shebangs (e.g. `#!/usr/bin/env`) aren't necessary since you're
+/// > already explicitly passing your script into the right shell.
```

Comment:
I would not include this. I think most users are just going to copy and paste and not wonder about it. If they do wonder about it, then they're probably able to figure it out themselves. But feel free to keep it if you think it'll help

Reply:
I think it's not harmful to include and might head off some questions before they hit issues or the Discord, so I'd like to keep it in